### PR TITLE
Project init

### PR DESCRIPTION
**Pull request by rozicd**

### CODE REVIEW

File: RBSBack/Controllers/UserController.cs [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+            }
+            else
+            {
+                return Ok(_user);
+            }
+        }
+
+
+        [HttpGet("{id}")]
```
Comment (MarkoErdelji) [inline RBSBack/Controllers/UserController.cs:143]: Can every type of user can use this endpoint?

File: RBSBack/Program.cs [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+builder.Services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
+    .AddJwtBearer(options =>
+    {
+        options.TokenValidationParameters = new TokenValidationParameters
+        {
+            ValidateIssuer = false,
+            ValidateAudience = false,
+            ValidateLifetime = true,
+            ValidateIssuerSigningKey = true,
+            IssuerSigningKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes("zanzibarrbszanzibarminirbszanzibarrbszanzibarminirbs"))
```
Comment (MarkoErdelji) [inline RBSBack/Program.cs]: This needs to be in enviroment variable

**Review by MarkoErdelji — APPROVED** [review]
Looks good!

**Review by ognjen34 — APPROVED** [review]
_**Brilliant**_

**Pull request merged**